docs/source/serving/service-yaml-spec.rst

```diff
@@ -44,16 +44,22 @@ Available fields:
         # use a fixed number of replicas (the same as min_replicas) and ignore
         # any QPS threshold specified below.
         max_replicas: 3
-        # Thresholds below describe when to scale up or down.
-        # QPS threshold for scaling up (optional). If the QPS of your service
-        # exceeds this threshold, SkyServe will scale up your service by one
-        # replica. If not specified, SkyServe will **NOT** scale up your service.
-        qps_upper_threshold: 10
-        # QPS threshold for scaling down (optional). If the QPS of your service
-        # is below this threshold, SkyServe will scale down your service by one
-        # replica. If not specified, SkyServe will **NOT** scale down your service.
-        qps_lower_threshold: 2
-
+        # Following specs describe the autoscaling policy.
+        # Target request per second per replica (optional). SkyServe will scale your
+        # service so that, ultimately, each replica manages approximately
+        # target_rps_per_replica queries per second. **Autoscaling will only be
+        # enabled if this value is specified.**
+        target_qps_per_replica: 5
```

Comment:
nit: should we call it `max_qps_per_replica`? I am a bit confused by the meaning of `target` here. Is it a widely used term in this case?

Reply:
@Michaelvll I think using `target` is probably fine. 
For example, Ray Serve autoscaler uses `target_num_ongoing_requests_per_replica`
https://docs.ray.io/en/latest/serve/advanced-guides/advanced-autoscaling.html